code/game/objects/effects/overlays.dm

```diff
@@ -132,6 +132,28 @@
 	duration = 8
 	randomdir = 0
 
+/obj/effect/overlay/temp/gib_animation
+	icon = 'icons/mob/mob.dmi'
+	duration = 15
+	randomdir = 0
```

Comment:
Technically unneeded if the sprite has no dirs; all the gib/dust animations have no dirs.

<sub>Also phil is using something I did aaaaaaa</sub>